index.html

```diff
@@ -828,33 +879,66 @@ <h3>Subjects</h3>
             <h3>Controllers</h3>
 
             <p>
-A [=controller=] is an entity that is authorized to make changes to a
-[=controller document=].
+A [=controller=] is any entity capable of making changes to a
+[=controller document=]. Whoever can update
+the content of the resource returned from dereferencing the controller
+document's canonical URL is, by definition, a controller of the
+document and its canonical identifier. Proofs that satisfy a
```

Comment:
That language doesn't say you can change a canonical identifier.

It says you establish control of the identifier by creating proofs.

>Whoever can update
the content of the resource returned from dereferencing the controller
document's canonical URL is, by definition, a controller of the
document and its canonical identifier.

This is correct. If you can update the content of the canonical resource, you are a controller of the document's canonical identifier.

Reply:
Well, again from my RDF-tainted view, I am not sure what it means to "control the identifier". Control means an ability to change; but changing the identifier means, strictly speaking, to _replace_ the document because a document and its identifier are intimately bound. (An RDF Resource is specified by its URL. Changing a URL means to introduce a different resource.)

But I acknowledge that this is very much an RDF view of things, so I won't lie down the road over this.

Reply:
@iherman 
>  am not sure what it means to "control the identifier"

Yes. This update is describing that it means to control the identifier.